src/bun.js/api/BunObject.zig

```diff
@@ -894,10 +893,11 @@ fn doResolve(globalThis: *JSC.JSGlobalObject, arguments: []const JSValue) bun.JS
         from_str,
         is_esm,
         false,
+        false,
     );
 }
 
-fn doResolveWithArgs(ctx: js.JSContextRef, specifier: bun.String, from: bun.String, is_esm: bool, comptime is_file_path: bool) bun.JSError!JSC.JSValue {
+fn doResolveWithArgs(ctx: js.JSContextRef, specifier: bun.String, from: bun.String, is_esm: bool, comptime is_file_path: bool, is_user_require_resolve: bool) bun.JSError!JSC.JSValue {
```

Comment:
nit: the 3 booleans would be nice as an options struct with default false. wonder if it's worth removing the comptime